### ISSUE TITLE
Q: "Screen DPI" vs physical font size

### ISSUE THREAD

**Issue by mergen3107** [koreader/koreader#12058]
I am sorry for asking this question again (I am pretty sure I did in the past).

Here is the workflow and questions I have right now:
1) I have 10-inch Kindle Scribe and 6-inch Kobo Clara HD on hand.
2) I have profiles with different font faces and their size - but all of them result in the same number of 45 lines per page on Kindle Scribe.
3) I would like to find suitable scaling of `screen size DPI` setting in `top menu - screen` so that I could use exact same profiles from Scribe on Clara HD. The result of this I imagine to be something like this:

<img src= "https://github.com/koreader/koreader/assets/6279855/5bf20c23-ee82-4adb-996e-f512c7dafa61" height="800">

Here the green rectangle is how many "effective lines" (without the progress bar) I expect from Kobo, while the actual text size (physical size with a ruler) stays the same.

Is there a factor I can calculate?
Or should I just take a ruler and take manual trial and error?
SpinWidget for screen DPI has a step of 10. If I override this in `settings.reader.lua` with something else, say, 123, will it work or break? :D 

Thanks!

**Comment by mergen3107** [comment]
Main idea is to preserve physical size of the fonts that I found to be comfortable for reading long-term on Scribe.

**Comment by mergen3107** [comment]
Thanks @zwim ! Totally forgot about this feature.
So I'll go and adjust screen dpi until they match :D

**Comment by mergen3107** [comment]
I found that setting 330 DPI on Kobo Clara HD makes the profiles from Kindle Scribe (120 DPI) work the same! The only thing left to adjust is the height of the progress bar and status bar. They are quite small, but OK (main text is more important for me).

Conversion factor of number of pages in a text-only book with the same style tweaks applied is 2.585.
For example, I tested it with a 836-page book on Scribe that translated into 2161 pages on Clara HD.

This was tested with only one font though.
